draft-ietf-quic-qpack.md

```diff
@@ -471,9 +471,10 @@ this MUST be treated as a connection error of type QPACK_ENCODER_STREAM_ERROR.
 ## Dynamic Table {#header-table-dynamic}
 
 The dynamic table consists of a list of field lines maintained in first-in,
-first-out order. Each HTTP/3 endpoint holds a dynamic table that is initially
+first-out order.  Each QPACK decoder holds a dynamic table that is initially
 empty.  Entries are added by encoder instructions received on the encoder
-stream; see {{encoder-instructions}}.
+stream; see {{encoder-instructions}}.  The encoder tracks the contents of the
+dynamic table maintained by the decoder.
```

Comment:
I think it's slightly odd to say that the encoder tracks the decoder, when in practice the encoder is doing the inserts and uses it for compression.  Let me think a bit.

Reply:
How about:

> The dynamic table consists of a list of field lines maintained in first-in, first-out order.  A QPACK encoder and decoder share a dynamic table that is initially empty. The encoder adds entries to the dynamic table and sends them to the decoder via instructions on the encoder stream; see {{encoder-instructions}}.   

I'm on the fence about adding another sentence about how the encoder tracks the decoder's current state via the decoder stream.

Reply:
I was trying to leave room for the nuance that the encoder can forget the contents of the table entries at any point, while the decoder is obliged to keep the whole thing around.  That's why it feels more accurate to say that the table "lives" at the decoder, and the encoder is tracking and manipulating it remotely.  But I can live with saying that they share it, which is also true.